### ISSUE TITLE
provide history of older "releases" of the codebase.

### ISSUE THREAD

**Issue by mathematicalmichael** [mindthegrow/cafelytics#24]
maybe we keep the original matlab code anyway? I don't know, perhaps put it on a separate branch with some mention of the archival process on the README. We can actually just refer people to the last commit number before the merge of #15 and provide some backstory in a blog post. 

That said I eventually do want auto-generated documentation and it would be nice if some markdown file from this repo just got incorporated into the docs, however that happens.

Part of this process should involve re-writing the README, but I'll raise another issue for that.

**Comment by djdebonis** [comment]
I do like this idea because it demonstrates the evolution of your software dev journey, Mind the Grow as a company, and the software itself.

**Comment by mathematicalmichael** [comment]
yeah but the beauty of git is that we can literally just hyperlink to the state of the project at any time in the past.

**Comment by mathematicalmichael** [comment]
todo:
- create a release for the major checkpoints
- create a changelog and link to the releases + git hashes from there

super low priority though.
